accounts/abi/unpackv2_test.go

```diff
@@ -0,0 +1,335 @@
+// Copyright 2015 The go-ethereum Authors
```

Comment:
What's with the `v2` stuff?

Reply:
I'm going to remove this file. Since this is a copy of the existing one, but uses the 'raw' unpacker, this is now not needed since they complex parsing happens on top of the 'raw' parsing.